core/templates/pages/license-page/license.mainpage.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html ng-app="oppia" itemscope itemtype="http://schema.org/Organization">
+  <head>
+    @load('base-components/header.template.html', {"title": "License Page | Oppia"})
+  </head>
+
+  <body>
+    <oppia-root>
+      <div ng-controller="Base" dir="<[direction]>">
+        <base-content>
+          <navbar-breadcrumb>
+          </navbar-breadcrumb>
+          <content>
+            <license-page></license-page>
+          </content>
+        </base-content>
```

Comment:
Are we expecting to have the oppia-footer at the end of this page?

Reply:
yes, I have added oppia-footer for license-page.